pkg/morph/event/notary_preparator.go

```diff
@@ -85,7 +86,7 @@ func notaryPreparator(localAcc util.Uint160, alphaKeys client.AlphabetKeys, bc B
 	contractSysCall := make([]byte, 4)
 	binary.LittleEndian.PutUint32(contractSysCall, interopnames.ToID([]byte(interopnames.SystemContractCall)))
 
-	dummyInvocationScript := append([]byte{byte(opcode.PUSHDATA1), 64}, make([]byte, 64)...)
+	dummyInvocationScript := slices.Concat([]byte{byte(opcode.PUSHDATA1), 64}, make([]byte, 64))
```

Comment:
Not needed.

Reply:
why? creating two slices and concatinating them looks better with slice.Concat that with appending, IMO. `make([]byte, 64)...` is a strange thing to see

Reply:
Not that it's important here, but https://github.com/nspcc-dev/neo-go/issues/3089#issuecomment-2308277770. `append` is shorter as well. Although here a single allocation can be made, but that'd be more code.

Reply:
ok, returned

Reply:
> append is shorter as well

i am reading it a little bit longer: "create a slice; extract its elements; their elements are the part of another slice now", while contactination is a simple well-known operation. not a problem, i just dreaming of some built int `concat` func (it wont be shorter with append then)